anago

```diff
@@ -585,12 +584,20 @@ update_github_release () {
 
   # post release data
   logecho "$release_verb the $RELEASE_VERSION_PRIME release on github..."
+  local changelog_url="$K8S_GITHUB_URL/blob/master/CHANGELOG.md"
+  local release_body=cat <<+
```

Comment:
I'm not sure there's a reason to break this out into $release_body.  I'm not going to argue the point heavily, but this stuff can be a bit sensitive.  Do be sure you run a full mocked run of this and the upload still works.